Gemfile

```
@@ -1,6 +1,6 @@
 source 'http://rubygems.org'
 
-ruby '2.5.5'
+ruby '2.7.0'
```

We're using Ruby 2.6.5 for this current cohort:

```suggestion
ruby '2.7.0'
```